src/installer/tests/Directory.Build.targets

```diff
@@ -92,6 +92,11 @@
     <RunSettingsTestCaseFilter>$(TestCaseFilter)</RunSettingsTestCaseFilter>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(IsTestProject)' == 'true'">
```

Comment:
IsTestProject is not the right thing. As this is MTP, use `IsTestingPlatformApplication` instead.